modules/fedimint-wallet-server/src/lib.rs

```diff
@@ -786,6 +803,22 @@ impl ServerModule for Wallet {
                     Ok(module.consensus_module_consensus_version(&mut context.dbtx().into_nc()).await)
                 }
             },
+            api_endpoint! {
+                ACTIVATE_CONSENSUS_VERSION_VOTING_ENDPOINT,
+                ApiVersion::new(0, 0),
```

Comment:
Probably not API version 0.0, but 0.1 or even higher?

Reply:
Yeah, this does not look right. We should bump the minor api version and put that version here. @joschisan